java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -266,6 +269,111 @@ CompletableFuture<ClusterValue<Map<String, Object>[]>> functionList(
      */
     CompletableFuture<String> functionDelete(String libName, Route route);
 
+    /**
+     * Returns the serialized payload of all loaded libraries.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/function-dump/">redis.io</a> for details.
+     * @return The serialized payload.
+     * @example
+     *     <pre>{@code
+     * ClusterValue<byte[]> data = client.functionDump().get();
+     * // data stores serialized dump from all primary nodes
+     * // it could be saved to restore loaded functions on any Redis instance
+     * }</pre>
+     */
+    CompletableFuture<ClusterValue<byte[]>> functionDump();
+
+    /**
+     * Returns the serialized payload of all loaded libraries.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/function-dump/">redis.io</a> for details.
+     * @param route Specifies the routing configuration for the command. The client will route the
+     *     command to the nodes defined by <code>route</code>.
+     * @return The serialized payload.
+     * @example
+     *     <pre>{@code
+     * byte[] data = client.functionDump(RANDOM).get().getSingleValue();
+     * // now data could be saved to restore loaded functions on any Redis instance
+     * }</pre>
+     */
+    CompletableFuture<ClusterValue<byte[]>> functionDump(Route route);
+
+    /**
+     * Restores libraries from the serialized payload.<br>
```

Comment:
might be slightly clearer although I know we already state this below, up to you. Same applies elsewhere in this PRs docs:
```suggestion
     * Restores libraries from the serialized payload returned by {@link #functionDump()}.<br>
```